src/models/Dismiss.ts

```diff
@@ -0,0 +1,16 @@
+export enum DismissPrompt {
+  Push = 'push',
+  Web  = 'web' // (non-push)
```

Comment:
Web, is to confusing here. Maybe channel?

Reply:
Decided on `NonPush` per discussion